python/tvm/relay/analysis/analysis.py

```diff
@@ -431,3 +431,23 @@ def get_calibration_data(mod, data):
         calib_data[gvar] = value
 
     return calib_data
+
+
+def extract_intermdeiate_expr(mod, expr_id):
+    """Extract Relay Expr by the local variable number
+
+    This function is used for extracting Relay Expr
+    by the local variable number of the main function
+    that we can see in `print(mod["main"])`.
```

Comment:
It would be nice to have an example here. `local variable number` is not a word in TVM, we should just refer to them as expression ID or something.